koekalenteri-frontend/src/config.default.js

```diff
@@ -0,0 +1,11 @@
+// You can obtain these values by running:
+// aws cloudformation describe-stacks --stack-name <YOUR STACK NAME> --query "Stacks[0].Outputs[]"
+
+const config = {
+    "aws_user_pools_web_client_id": "AWS_USER_POOLS_WEB_CLIENT_ID",     // CognitoClientID
+    "api_base_url": "API_BASE_URL",                                     // KoekalenteriunctionApi
+    "cognito_hosted_domain": "COGNITO_HOSTED_DOMAIN",                   // CognitoDomainName
```

Comment:
Codacy has a fix for the issue: [Strings must use singlequote.](https://app.codacy.com/gh/koekalenteri/koekalenteri/pullRequest?prid=7376473)
```suggestion
    'cognito_hosted_domain': "COGNITO_HOSTED_DOMAIN",                   // CognitoDomainName
```